lib/galaxy/util/__init__.py

```diff
@@ -2029,8 +2029,5 @@ def to_content_disposition(target: str) -> str:
 def validate_doi(doi: str) -> bool:
     if len(doi) > DOI_MAX_LENGTH:
         return False
-    prefix = "https://doi.org/|doi.org/|doi:"
-    doi_prefix = r"10\.\d+"
-    doi_suffix = r"\S+"
-    doi_re = re.compile(f"^{prefix}{doi_prefix}/{doi_suffix}$")
-    return bool(doi_re.match(doi))
+    doi_re = re.compile(r"(10\.(\d)+\/(.+))")
+    return bool(doi_re.search(doi))
```

Comment:
`match` -> `search`

Reply:
xref: https://github.com/galaxyproject/galaxy/pull/20033/files#r2104529118

Also, I found this to be helpful: https://www.crossref.org/blog/dois-and-matching-regular-expressions/

Reply:
I think this would result in many more false positives. This part `.+` especially - i.e., this would match: 
`10.1/just some text`

Reply:
`10.1/just some text` is technically a valid DOI but I am fine with drawing line before there